common/fileutil.go

```diff
@@ -333,31 +332,37 @@ func CopyFile(source, destination string) {
 	ErrCheckExitf(err, 1, "error copying from source %s to destination file %s: %s", source, destination, err)
 }
 
-func BaseName(filename string) string {
-	return filepath.Base(filename)
-}
-
-func DirName(filename string) string {
-	return filepath.Dir(filename)
-}
-
-func AbsolutePath(value string) string {
-	filename, err := filepath.Abs(value)
-	ErrCheckExitf(err, 1, "error getting absolute path for %s", value)
-	return filename
-}
-
-func Mkdir(dirName string) {
-	err := os.Mkdir(dirName, PublicDirectoryAttr)
-	ErrCheckExitf(err, 1, "error creating directory %s\n%s\n", dirName, err)
-}
-
-func Rmdir(dirName string) {
-	err := os.Remove(dirName)
-	ErrCheckExitf(err, 1, "error removing directory %s\n%s\n", dirName, err)
-}
+// These lines should:
+// 1) Do not exit. Return errors instead.
+// 2) Wrappers for 1 single line will be removed to use errors instead
+// 3) I am commenting out these lines just to force a failure wherever they are being called. That way
+//    will be easier to detect all the places where they should be changed/removed.
```

Comment:
I have taken a different approach. I believed that single sandbox was the priority, and thus I removed or changed all the functions that could exit from `sandbox/sandbox.go`.
I have done that only partially for replication, multiple, and group replication. 
The plan is to continue there if the changes for single sandbox are satisfactory.

As for the method, using [Goland](https://www.jetbrains.com/shop/eform/opensource) the refactoring is fairly easy.